base/files/file_starboard.cc

```diff
@@ -181,8 +196,9 @@ int File::WriteAtCurrentPos(const char* data, int size) {
     return -1;
 
   SCOPED_FILE_TRACE_WITH_SIZE("WriteAtCurrentPos", size);
-
-  return SbFileWriteAll(file_.get(), data, size);
```

Comment:
Having //base/files instrumented is fine, but we mostly use https://github.com/youtube/cobalt/blob/main/starboard/common/file.cc. Search for ScopedFile.

The complication around //starboard/common is that it is used by both Cobalt and Starboard. 
So either guard the instrumentation by #ifndef STARBOARD_IMPLEMENTATION or rely on the StatsTrackerUndefined.

We also have various places where we directly use SbFileWrite or SbFileAtomicReplace.

Reply:
> Search for ScopedFile

Uh, i totally didn't notice that. Why are we using that instead of base::File from above-Base code ? It's almost identical in intent.